File: app/src/main/java/com/music/player/bhandari/m/customViews/MyScrollView.java
package com.music.player.bhandari.m.customViews;

import android.content.Context;
import android.util.AttributeSet;
import android.widget.ScrollView;

/**
 * scroll view for setting fading only from bottom
 * used in artist info page for fading content under playback controls
 */

public class MyScrollView extends ScrollView {
    public MyScrollView(Context context) {
        super(context);
    }

    public MyScrollView(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    public MyScrollView(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }

    @Override
    protected float getTopFadingEdgeStrength() {
        return 0;
    }
}

File: app/src/main/java/com/music/player/bhandari/m/customViews/MyScrollView.kt
package com.music.player.bhandari.m.customViews

import android.content.Context
import android.util.AttributeSet
import android.widget.ScrollView

/**
 * scroll view for setting fading only from bottom
 * used in artist info page for fading content under playback controls
 */
class MyScrollView : ScrollView {
    constructor(context: Context?) : super(context)
    constructor(context: Context?, attrs: AttributeSet?) : super(context, attrs)
    constructor(context: Context?, attrs: AttributeSet?, defStyleAttr: Int) : super(context, attrs, defStyleAttr)

    override fun getTopFadingEdgeStrength(): Float {
        return 0F
    }
}